src/components/search/SearchButton.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function SearchButton() {
```

Comment:
Align your code to use arrow function. See this link for the benefits of using arrow function. https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/